src/__tests__/lib/containers/entity_finder/details/DetailsView.test.tsx

```diff
@@ -168,32 +173,35 @@ describe('DetailsView tests', () => {
     })
 
     describe('Determines correct row appearance', () => {
-      it('Creates a row with the default appearance', () => {
+      it('Creates a row with the default appearance', async () => {
```

Comment:
In many places, replace synchronous `getBy` with async `findBy` to get rid of many "not wrapped in act" warnings.